src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -112,26 +105,35 @@ public static Export CreateExport<T>(object exportInstance, string contractName
                 Console.WriteLine($"Removing export: {exportToRemove.Definition.ContractName}");
                 subsetOfRequiredExports.RemoveAt(i);
 
-                Action act = () => TryCompose(importer, catalog, subsetOfRequiredExports);
+                Action act = () => Compose(importer, typeof(TTypeToCheck), subsetOfRequiredExports.ToArray());
                 act.Should().Throw<CompositionException>();
             }
 
             // Finally try importing when all of the required exports are available -> success
-            TryCompose(importer, catalog, requiredExports);
+            Compose(importer, typeof(TTypeToCheck), requiredExports);
         }
 
-        private static void TryCompose(object importer, TypeCatalog catalog, IEnumerable<Export> exports)
+        /// <summary>
+        /// Performs a MEF compose with the supplied objects
+        /// </summary>
+        /// <param name="importer">The instance that is importing the exported type</param>
+        /// <param name="typeToExport">The type to of export being tested</param>
+        /// <param name="additionalExports">Any other exports that are required. Can be empty.</param>
+        public static void Compose(object importer, Type typeToExport, params Export[] additionalExports)
```

Comment:
Modified the existing method slightly so it is useful as a `public` function that can be used from the `ImportMany` tests.

I moved creating the `TypeCatalog` into this method so the callers don't need to know about it.

I also changed the `additionalExports` argument to a `params` array to make it easier to consume (which is why the other methods in this class had to change to pass an array rather than an `IEnumerable`).